Formula/hubble.rb

```diff
@@ -0,0 +1,22 @@
+require "open3"
+
+class Hubble < Formula
+  desc "Network, Service & Security Observability for Kubernetes using eBPF"
+  homepage "https://github.com/cilium/hubble"
+  url "https://github.com/cilium/hubble/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "ebb54783c57e7997986dcc5a73f4f845015920b857dd348abd7a02e5d3af65ec"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "hubble"
+    bin.install "hubble"
+  end
+
+  test do
+    _, _, status = Open3.capture3("#{bin}/hubble", "version")
+
+    assert_equal 0, status
```

Comment:
Let's avoid using `open3`, we can use `shell_output` instead. Also, we need a functional test, as version and help output comparisons almost always fail to detect broken installs. If this requires a running/detectable service or cluster, you could try to connect with invalid credentials or details and confirm that it fails as expected.